addons/io_scene_gltf2_msfs/blender/msfs_material_function.py

```diff
@@ -251,7 +251,8 @@ def defaultShadersTree(self):
             width = 200.0,
             frame = baseColorFrame
         )
-        blendColorMapNode.inputs[0].default_value = 1.0
+        #Input Factor
+        blendColorMapNode.inputs[self.inputs0].default_value = 1.0
```

Comment:
Where do you set self.input0 ?

Reply:
The self.input0 is a fix for my own version to make use of the newer nodes Mix node replacing MixRGB in 3.6 and 4.X+ - I removed that in the commit b664.